README.md

```diff
@@ -1 +1,22 @@
-# postgresql-watcher
\ No newline at end of file
+# postgresql-watcher
+
+
+Casbin role watcher to be used for monitoring updates to casbin policies
+## Installation
+```bash
+pip install postgresql-watcher
+```
+
+## Basic Usage Example 
+### With Flask-authz
+```python
+from flask_authz import CasbinEnforcer
+from postgresql_watcher import PostgresqlWatcher
+from flask import Flask
+from casbin.persist.adapters import FileAdapter
+
+casbin_enforcer = CasbinEnforcer(app, adapter)
+watcher = PostgresqlWatcher(host=HOST, port=PORT, user=USER, password=PASSWORD)
+watcher.set_update_callback(casbin_enforcer.e.load_policy())
```

Comment:
The update callback is set here, which according to my understanding should be invoked if this watcher instance is notified that another enforcer has changed it's policies. Where is this being invoked though, as it doesn't seem to be called anywhere?